registration/src/main/proto/lobby/registration/events.proto

```diff
@@ -21,12 +21,16 @@ syntax = "proto3";
 
 package lobby.registration.event;
 
-import "lobby/registration/order.proto";
-
 option java_package="com.lobby.registration.event";
 option java_outer_classname = "RegistrationEventsProto";
 option java_multiple_files = true;
 
-message OrderCreated {
+import "lobby/registration/order.proto";
+import "lobby/registration/conference.proto";
+
+message RegisteredToConference {
```

Comment:
Please document commands and events. Describe when each event occurs from the business point of view.